README.md

```diff
@@ -1,2 +1,21 @@
 # hw01
 Repository for homework 1
+test
+
+	
+# Header 1
+
+My name is **Jason** and I *love* frisbee and *CS*. I have never used Github but this is a cool photo ![GitHub Logo](/images/logo.png)
+Format: ![Alt Text](url).
```

Comment:
This part is a little funky when rendered.  I think it would have been best to either save the image in the repo or do what you did below.